src/database/migrations/0065-alter-application_form_fields-add-instructions.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE application_form_fields
+ADD COLUMN instructions text;
+
+COMMENT ON COLUMN application_form_fields.instructions IS
+'Instructions (supplementing or in addition to the label) for '
+'the user to fill information into the application form field.';
```

Comment:
Missing a newline (do you have editorconfig properly enabled on your IDE? it should prevent this from happening / insert newlines on save)